packages/flutter/lib/src/material/typography.dart

```diff
@@ -79,26 +79,6 @@ enum ScriptCategory {
 ///  * <https://material.io/design/typography/>
 @immutable
 class Typography with Diagnosticable {
-  /// Creates a typography instance.
-  ///
-  /// This constructor is identical to [Typography.material2014]. It is
-  /// deprecated because the 2014 material design defaults used by that
-  /// constructor are obsolete. The current material design specification
-  /// recommendations are those reflected by [Typography.material2018].
-  @Deprecated(
-    'The default Typography constructor defaults to the 2014 material design defaults. '
-    'Applications are urged to migrate to Typography.material2018(), or, if the 2014 defaults '
-    'are desired, to explicitly request them using Typography.material2014(). '
-    'This feature was deprecated after v1.13.8.',
-  )
-  factory Typography({
-    TargetPlatform? platform,
-    TextTheme? black,
-    TextTheme? white,
-    TextTheme? englishLike,
-    TextTheme? dense,
-    TextTheme? tall,
-  }) = Typography.material2014;
```

Comment:
It seems odd to have no Typography constructor called "Typography".  Perhaps just change ` = Typography.material2014` to ` = Typography.material2018`?